src/operator/nn/mkldnn/mkldnn_fully_connected-inl.h

```diff
@@ -80,6 +86,39 @@ struct MKLDNNFCFullParam {
   std::vector<float> output_scales = {0.0f};
 };
 
+// Forward declaration
+class FCInputIndex {
```

Comment:
Why not put the full implementation of FCInputIndex here?

Reply:
Ok.